src/controllers/EscapePodController.test.ts

```diff
@@ -204,5 +204,56 @@ describe(
 
     );
 
+    it(
+      "deleteById should return 204 status on success", async () => {
+
+        // Given
+
+        const fakeService = sinon.createStubInstance(EscapePodService);
+        const fakeReq = getRequestMock({ params: { id: "1" } });
+        const fakeRes = getResponseMock();
+        const controller = new EscapePodController(fakeService);
+
+        // When
+
+        fakeService.deleteById.throws(new NotFoundException());
+
+        // Then
+
+        await expect(controller.deleteById(
+          fakeReq, fakeRes as any, null,
+        )).rejects.toThrow(NotFoundException);
```

Comment:
The test's title says return 204 status on success, but we are testing a whole different things (i.e., not found exception).